sdk/identity/azure-identity/src/main/java/com/azure/identity/ChainedTokenCredentialBuilder.java

```diff
@@ -44,6 +45,19 @@ public ChainedTokenCredentialBuilder addLast(TokenCredential credential) {
         return this;
     }
 
+    /**
+     * Adds all of the credentials in the specified collection at the end
+     * of this chain, as if by calling {@link ChainedTokenCredentialBuilder#addLast(TokenCredential)} on each one,
+     * in the order that they are returned by the collection's iterator.
+     *
+     * @param credentials the collection of credentials to be appended to the chain.
+     * @return An updated instance of the builder.
+     */
+    public ChainedTokenCredentialBuilder addAll(Collection<? extends TokenCredential> credentials) {
```

Comment:
Can this be a List instead?
https://github.com/Azure/azure-sdk-for-java/pull/12203#discussion_r440485654

Reply:
as, discussed, keeping it to Colleciton.